examples/cfd/ldc_pinns/config.yaml

```diff
@@ -0,0 +1,33 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 - 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+arch:
+  decoder:
+    out_features: 3
+    layers: 1
+    layer_size: 32
+
+  fno:
+    in_channels: 2
+    dimension: 2
+    latent_channels: 32
+    fno_layers: 4
+    fno_modes: 12
+    padding: 9
+
+scheduler:
+  initial_lr: 1.E-3
+  decay_rate: .9995
```

Comment:
Can the type of gradient calculation used be part of the config?

Reply:
It is possible. But since this is an introductory example, I decided not to do that. The idea was to provide a minimal code with a focus on the utilities. For FD and Least Squares, the dataloader pipeline will have to be different and enabling that via config can become a bit verbose.